src/EditorFeatures/DiagnosticsTestUtilities/NamingStyles/NamingStylesTestOptionSets.cs

```diff
@@ -220,7 +220,7 @@ private static NamingStylePreferences FieldNamesAreCamelCaseWithUnderscorePrefix
             return info;
         }
 
-        private static NamingStylePreferences FieldNamesAreCamelCaseWithFieldUnderscorePrefixOption()
+        internal static NamingStylePreferences FieldNamesAreCamelCaseWithFieldUnderscorePrefixOption()
```

Comment:
Why not pass `.editorconfig` to tests instead of making these `internal`?

This will improve coverage for naming styles parser as well.

Reply:
Laziness 😁